src/main/java/mate/academy/hibernate/relations/dao/impl/ActorDaoImpl.java

```diff
@@ -12,11 +15,29 @@ public ActorDaoImpl(SessionFactory sessionFactory) {
 
     @Override
     public Actor add(Actor actor) {
-        return null;
+        Session session = factory.openSession();
+        if (session == null) {
+            throw new DataProcessingException("Session not initialized");
+        }
```

Comment:
Checking if the session is null is unnecessary here. The `openSession()` method should return a valid session or throw an exception if it cannot do so. You can remove lines 19-21.